week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,97 @@
+//add event listener to blue button
+
+let blueBtn = document.getElementById('blueBtn');
+//console.log(blueEl);
+
+let jumboEl = document.querySelector('.jumbotron');
+//console.log(jumboEl);
+
+let donateBtn = document.getElementById('donateBtn');
+
+let volunteerBtn = document.getElementById('volunteerBtn');
+
+
+
+function onClickBlueBtn(){
```

Comment:
I'd think about the name of this function in terms of the action it performs. When the user clicks on the button it triggers an event and this function will change the styling to a "blue theme". But am just being picky here 😁